src/Health/HealthController.ts

```diff
@@ -167,6 +173,12 @@ export class HealthController {
     )
   }
 
+  public purgeIpfsDirectoryHashByTransactionIds = pipeP(
+    this.log(LogTypes.trace)('purging IPFSDirectoryHash for transactionIds'),
+    this.removeIPFSDirectoryHashByTransactionId,
```

Comment:
Why not directly do `this.ipfsDirectoryHashDAO.deleteByTransactionIds` here instead?

Reply:
I'm following a pattern in the controller to log elements. Are we changing the pattern?

Reply:
Got it. Let's not